tdp/conftest.py

```diff
@@ -1,17 +1,48 @@
 # Copyright 2022 TOSIT.IO
 # SPDX-License-Identifier: Apache-2.0
 
+import logging
+import pytest
 from pathlib import Path
-from typing import Mapping
+from typing import Generator, Mapping
 
 import yaml
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session, sessionmaker
 
 from tdp.core.collection import (
     DAG_DIRECTORY_NAME,
     DEFAULT_VARS_DIRECTORY_NAME,
     PLAYBOOKS_DIRECTORY_NAME,
     SCHEMA_VARS_DIRECTORY_NAME,
 )
+from tdp.core.models import Base
+
+
+sqlalchemy_logger = logging.getLogger("sqlalchemy.engine")
+sqlalchemy_logger.setLevel(logging.INFO)
```

Comment:
Where this logger is used ?

Reply:
It is not used in my code, it is a way to define log level as "INFO" for sqlalchemy in those tests.

We could replace it with:

```py
logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
```

Reply:
ok so no need to put it in a variable.

Reply:
Beyond the syntax, idk if we should keep it.

Reply:
I removed it.